### ISSUE TITLE
Unable to log in, even after resetting password

### ISSUE THREAD

**Issue by LSN57** [madcowfred/padherder-issues#360]
I'm unable to log into padherder from my home computer, from both Chrome and Firefox, even after having a password reset link sent to the email I'm trying to log in with and resetting the password via the link in the email.

Not sure how to send you the email I'm trying to use without it being publicly viewable here, but my original username is LSN, and then I also tried signing up again with username LSN57 because I wasn't able to log into my original account. I think they use the same email address.

**Comment by madcowfred** [comment]
I just tested the reset process by borrowing your account (temporarily changed email address to my own) and it worked fine, not sure what to tell you. If you're receiving the reset email and following the steps it should work.

**Comment by LSN57** [comment]
Still couldn't log in, from separate computers, using Chrome and Firefox on one of them. Could it be that now that I've used the same email address (starts with zxbo) for both accounts, it's getting confused about which one to log me in as? I did try to reset my password again, and it sent the email twice, once for each account. I reset both accounts to the same password, but still couldn't log in (this is where I also tried Firefox).

**Comment by madcowfred** [comment]
Are you trying to use your email address in the username field instead of the username? That's all I can think of :|

**Comment by LSN57** [comment]
You know what...that might be it. Oops.

I had set my home computer to save my login details, so every time I went
to padherder.com, it auto-filled the Username and Password inputs on every
page, but my email was in the Username field, and I never noticed that it
was supposed to be Username and not email. I'll verify and update my ticket
later when I get back to my regular computer, but from my work computer I
logged in alright...
